cmd/celestia/internal/admin/admin.go

```diff
@@ -1,69 +1,59 @@
-package main
+package admin
 
 import (
-	"context"
 	"errors"
 	"strings"
 
 	"github.com/filecoin-project/go-jsonrpc/auth"
 	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-node/cmd/celestia/internal"
 )
 
 func init() {
-	nodeCmd.AddCommand(nodeInfoCmd, logCmd, verifyCmd, authCmd)
-	rootCmd.AddCommand(nodeCmd)
-}
+	NodeCmd.AddCommand(nodeInfoCmd, logCmd, verifyCmd, authCmd)
 
-var nodeCmd = &cobra.Command{
-	Use:   "node [command]",
-	Short: "Allows administrating running node.",
-	Args:  cobra.NoArgs,
-	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-		rpcClient, err := newRPCClient(cmd.Context())
-		if err != nil {
-			return err
-		}
+	NodeCmd.PersistentFlags().StringVar(
+		&internal.RequestURL,
+		"url",
+		"http://localhost:26658",
```

Comment:
i'm noticing some drift around defaults/settings and their usage. I think at a minimum we should 

a) move this to a const at the top vs inline
b) even better, create either commands/config/defaults.go or even a repo level config somewhere with a defaults.go

Reply:
I'd suggest creating a flag in one place and assigning it to all cmds